modules/machineset-creating.adoc

```diff
@@ -0,0 +1,261 @@
+  name: <cluster_name>-<machine_label>-<AWS-availability-zone> <3> <4>
+  namespace: openshift-cluster-api
+  resourceVersion: "9249" <1>
+  selfLink: /apis/cluster.k8s.io/v1alpha1/namespaces/openshift-cluster-api/machinesets/<cluster_name>-<machine_label>-<AWS-availability-zone> <1>
+  uid: 59ba0425-313f-11e9-861e-0a18047f0a28 <1>
+----
+<1> Remove this line.
+<2> Do not change the `<cluster_name>`.
+<3> For each `<machine_label>` instance, specify the name of the new MachineSet.
+<4> Ensure that the AWS availability zone is correct in each instance of the
+`<AWS-availability-zone>` parameter.
++
+The `metadata` section resembles the following YAML:
++
+[source,yaml]
+----
+metadata:
+  labels:
+    sigs.k8s.io/cluster-api-cluster: <cluster_name>
+    sigs.k8s.io/cluster-api-machine-role: <new_machine_label>
+    sigs.k8s.io/cluster-api-machine-type: <new_machine_label>
+  name: <cluster_name>-<new_machine_label>-<AWS-availability-zone>
+  namespace: openshift-cluster-api
+----
+
+. In `<file_name>.yaml`, delete the `status` stanza:
++
+[source,yaml]
+----
+status:
+  availableReplicas: 1
+  fullyLabeledReplicas: 1
+  observedGeneration: 1
+  readyReplicas: 1
+  replicas: 1
+----
+
+. In `<file_name>.yaml`, update both instances of the `sigs.k8s.io/cluster-api-machineset` parameter
+values in the `spec` section to match the `name` that you defined in the `metadata` section:
++
+[source,yaml]
+----
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      sigs.k8s.io/cluster-api-cluster: cluster_name
+      sigs.k8s.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone> <1>
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        sigs.k8s.io/cluster-api-cluster: <cluster_name>
+        sigs.k8s.io/cluster-api-machine-role: <machine_label>
+        sigs.k8s.io/cluster-api-machine-type: <machine_label>
+        sigs.k8s.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone> <1>
+...
+----
+<1> Ensure that both the `sigs.k8s.io/cluster-api-machineset` parameter values
+match the `name` that you defined in the `metadata` section.
+
+. In `<file_name>.yaml`, add the node label definition to the spec. The label
+definition resembles the following stanza:
++
+[source,yaml]
+----
+  spec:
+    metadata:
+      labels:
+        node-role.kubernetes.io/<label_name>: "" <1>
+----
+<1> In this definition, `<label_name>` is the node label to add. For example, to
+add the `infra` label to the nodes, specify `node-role.kubernetes.io/infra`.
++
+The updated `spec` section resembles this example:
++
+[source,yaml]
+----
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      sigs.k8s.io/cluster-api-cluster: cluster_name
+      sigs.k8s.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone>
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        sigs.k8s.io/cluster-api-cluster: <cluster_name>
```

Comment:
If you were making a new machineset, would you update the parameters in this section to use the new label?